### PR TITLE
Use full path for appbase in dotnet.exe

### PR DESCRIPTION
**Pull request by ChengTian**
@davidfowl

### CODE REVIEW

**Comment by davidfowl** [comment]
@anurse can you move those files to be changed only on the CI, it's pretty annoying

**Comment by davidfowl** [comment]
@ChengTian add those files to teh gitignore

File: src/dotnet/dotnet.cpp [hunk 1]
```diff
@@ -294,6 +294,21 @@ int CallFirmwareProcessMain(int argc, wchar_t* argv[])
...
+    DWORD dwFullAppBase = GetFullPathNameW(data.applicationBase, MAX_PATH, szFullAppBase, nullptr);
+    if (!dwFullAppBase)
+    {
+        ::wprintf_s(L"Failed to get full path of application base: %S\r\n", data.applicationBase);
+        goto Finished;
+    }
+    else if (dwFullAppBase > MAX_PATH)
+    {
+        ::wprintf_s(L"Full path of application base is too long\r\n");
+        goto Finished;
```
Comment (davidfowl) [inline src/dotnet/dotnet.cpp]: Whats the exit code in this case? non 0?

Comment (ChengTian) [inline src/dotnet/dotnet.cpp]: Fixed. Returns 1 in this case

**Comment by ChengTian** [comment]
Rebased on top of @anurse 's change to get rid of annoying resource file changes. Also reverted resource files changes.

**Comment by davidfowl** [comment]
:shipit:

**Pull request merged**